frontend/src/modules/history/actions.js

```diff
@@ -11,18 +11,38 @@ import { actions as pluralActions } from 'core/plural';
 import { actions as resourceActions } from 'core/resource';
 import { actions as statsActions } from 'core/stats';
 
-import type { Entity } from 'core/api';
+import type { Entity, TranslationComment } from 'core/api';
 import type { Locale } from 'core/locale';
 
 export const RECEIVE: 'history/RECEIVE' = 'history/RECEIVE';
 export const REQUEST: 'history/REQUEST' = 'history/REQUEST';
 export const UPDATE: 'history/UPDATE' = 'history/UPDATE';
 
+export type HistoryTranslation = {|
+    +approved: boolean,
+    +approvedUser: string,
+    +date: string,
+    +dateIso: string,
+    +fuzzy: boolean,
+    +pk: number,
+    +rejected: boolean,
+    +string: string,
+    +uid: ?number,
+    +unapprovedUser: string,
+    +machinerySources: string,
+    +user: string,
+    +username: string,
+    +userGravatarUrlSmall: string,
+    +comments: Array<TranslationComment>,
+|};
```

Comment:
I moved this type from `reducer` to avoid circular imports. And it seems that this is the better way looking at other modules.

But this is mostly me guessing.